### PR TITLE
Remove modules depending on ethereum

### LINKED ISSUE #81: remove ethereum dependencies
remove ethereum dependencies and add default coder.

### PR DESCRIPTION
**Pull request by syuhei176**
We moved modules depending on Main chain to [wakkanay-ethereum](https://github.com/cryptoeconomicslab/wakkanay-ethereum).
removing ethereum dependencies
adding JsonCoder as default Coder

Close #81

### CODE REVIEW

File: src/coder/index.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
+export * from './Coder'
+export * from './Error'
 import EthCoder from './EthCoder'
-
-// TODO: change export based on environmental variable
-const Coder = EthCoder
-export default Coder
+export default EthCoder
```
Comment (syuhei176) [inline src/coder/index.ts]: I think we need a default coder for deciders' test code. @tkmct

Comment (tkmct) [reply src/coder/index.ts]: oh, are we going to extract coders into `wakkanay-ethereum`?
then we can make default coder using `JSON.stringify`.

Comment (syuhei176) [reply src/coder/index.ts]: > oh, are we going to extract coders into wakkanay-ethereum?
yep, I supposed to.
OK, I'll add json coder.

File: src/coder/index.ts [hunk 2]
```diff
@@ -1,5 +1,4 @@
-import EthCoder from './EthCoder'
-
-// TODO: change export based on environmental variable
-const Coder = EthCoder
-export default Coder
+export * from './Coder'
+export * from './Error'
+import JsonCoder from './JsonCoder'
+export default JsonCoder
```
Comment (syuhei176) [inline src/coder/index.ts]: How do we inject another Coder?

File: src/coder/JsonCoder.ts [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      })
+    d.setData(data)
+  } else {
+    throw JsonDecodeError.from(d)
+  }
+  return d
+}
+
+// JSON coder object
+const JsonCoder: Coder = {
```
Comment (syuhei176) [inline src/coder/JsonCoder.ts]: adding JsonCoder

File: src/coder/index.ts (hunk 2 above)
Comment (tkmct) [reply src/coder/index.ts]: we'll figure it out.

**Comment by tkmct** [comment]
need to write tests for verifiers.

File: __tests__/verifiers/Ed25519.test.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import {
```
Comment (syuhei176) [inline __tests__/verifiers/Ed25519.test.ts]: added test code for verifiers.

**Pull request merged**